locales/en-US.json

```diff
@@ -44,7 +44,8 @@
       "empty_street_name_labels": [
         "the walkway",
         "the cycleway",
-        "the mountain bike trail"
+        "the mountain bike trail",
+        "the pedestrian crossing"
```

Comment:
Thinking the en-US (the default) phrase should be `"the crosswalk"` as per: https://en.wikipedia.org/wiki/Pedestrian_crossing